lib/flatpak-game.lua

```diff
@@ -65,7 +65,6 @@ function identify(file)
 end
 
 function verify_gog(file)
-    print('Verifying game file.')
```

Comment:
Again, I don't need to see the stream of thought, so please merge any changes that you're correcting later directly in the original patch where you're modifying it.